.github/workflows/release-docker.yaml

```diff
@@ -31,6 +32,7 @@ jobs:
     if: ${{ contains(needs.version.outputs.number, '.') }}
     uses: ./.github/workflows/build-and-dockerize.yaml
     with:
+      ref: refs/tags/${{ inputs.tag }}
```

Comment:
I think this is not necessary, since `main.yml` invokes `release-stable.yml` which invokes `release-docker.yml` - which means the ref should be the same as the tagged version?